src/stories/containers/Finances/api/queries.ts

```diff
@@ -16,14 +17,63 @@ export const AllBudgets = () => ({
         description
       }
     }
-  `,
-});
+  `;
 
-export const fetchBudgetsFinances = async (): Promise<Budget[]> => {
-  const { query } = AllBudgets();
   const res = await request<{
     budgets: Budget[];
   }>(GRAPHQL_ENDPOINT, query);
 
   return res.budgets;
 };
+
+export const fetchAnalytics = async (
+  granularity: 'total' | 'annual' | 'semiAnnual' | 'quarterly' | 'monthly' | 'weekly' | 'daily' | 'hourly',
```

Comment:
The granularity should be a type in the `analytics` inteface and reused here to avoid to repeat all the types everytime

Reply:
Done! check: https://github.com/makerdao-ses/ecosystem-dashboard/pull/1117/commits/dcb1808c34b8389cc547be03221a06f17d27209d